docs: PEM-1296 removed web shell language (#1182)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/12-palette-webctl.md

<WarningBox>

If you are using [OIDC](/clusters/cluster-management/cluster-rbac#userbacwithoidc) with your host cluster, you will need the kubelogin plugin. Refer to the kubelogin GitHub repository [README](https://github.com/int128/kubelogin#setup) for installation guidance.

</WarningBox>

# Set up Kubectl

1. Log in to [Palette](https://spectrocloud.com).

2. Navigate to the left **Main Menu** and select **Clusters**.

3. Select the host cluster you want to access.

4. From the cluster overview page, navigate to the middle column containing cluster details and locate the **Kubernetes Config File** row.

5. Click on the kubeconfig link to download the file.

![Arrow pointing to the kubeconfig file](/clusters_cluster-management_palette-webctl_cluster-details-overview.png)

<br />

6. Open a terminal window and set the `KUBECONFIG` environment variable to the file path of the **kubeconfig** file.
  
  Example:
  ```shell
  export KUBECONFIG=~/Downloads/dev-cluster.kubeconfig 
  ```

You can now issue kubectl commands against your host cluster.

# Validation

> You can validate you have access to your cluster by attempting to issue kubectl commands against your host cluster. 

Reviewer: ```suggestion
Verify you have access to your host cluster by issuing kubectl commands against it. 
```
